### PR TITLE
fix VIP missing on map

### PR DESCRIPTION
**Pull request by elKei24**

### CODE REVIEW

**Review by maxammann — CHANGES_REQUESTED** [review]
Did you ask him whether he is fine with that? :sweat_smile:

**Comment by elKei24** [comment]
Of course not 😅

**Comment by RoAnBu** [comment]
:joy:

File: docker/reverse_proxy/www/sprites/osm-liberty.json [hunk 1]
```diff
@@ -1748,5 +1748,12 @@
     "width": 21,
     "x": 231,
     "y": 198
+  },
+  "knappo": {
```
Comment (aiquita) [inline docker/reverse_proxy/www/sprites/osm-liberty.json:1752]: Ohh, ich bin großer Fan :)

**Review by aiquita — APPROVED** [review]

**Comment by elKei24** [comment]
@maxammann face now consists of 8×8 pixels and I added a matte, so I think we are safe now

**Pull request closed without merging**